python/tox.ini

```diff
@@ -23,7 +23,10 @@ usedevelop = true
 deps =
     coverage
     mock
+    numpy
     pytest
+    mmh3
+extras = dev
```

Comment:
This just installs the `[dev]` stuff listed in the setup.py right? I don't think we need this here since we define the test dependencies right above this.